### ISSUE TITLE
Exported Game is not showing ImageTexture

### ISSUE THREAD

**Issue by AlexHolly** [godotengine/godot#20771]
I made an example maybe someone can test to confirm it is not the same as #20641

Tested with e9c083ad62b843e64d20ca04d05d0f3858016cfa

[New Game Projecte.zip](https://github.com/godotengine/godot/files/2270892/New.Game.Projecte.zip)

1. The sprite name is not the same
2. Run in Editor works fine
3. Run Exported Gamed(tested on x11, html5) shows only an error that test.png can't be found.

```
ERROR: load_image: Error opening file: res://sprites/test.png
   At: core/io/image_loader.cpp:55.
ERROR: texture_get_data: Condition ' !texture->active ' is true. returned: Ref<Image>()
   At: drivers/gles3/rasterizer_storage_gles3.cpp:1008.

```

Maybe it is related to https://github.com/godotengine/godot/issues/20641 but I don't see any name conflict.
I tried a few commits like ~1 or 2 months old, same issue. Feels like I am doing something wrong.

Tested Godot Stable 3.0.6 same issue.

Edit:
Looks like the exported version tries to load the .png and not the imported .stex file/path
https://github.com/godotengine/godot/blob/master/core/io/image_loader.cpp#L55

Edit2:
```
extends Control

onready var tr_field = $"TextureRect"

func _ready():
	var field = ImageTexture.new()
	field.load("res://sprites/test.png")
	tr_field.set_texture(field)
```

Edit3: So the solution was simple reduz helped me out.
1. Set "Import as" to "Image" 
2. Do not use (ImageTexture/StreamTexture).load(..) 
3. Use load(...)

**Comment by tomasantunes** [comment]
please see issue #20641

**Comment by AlexHolly** [comment]
I made an example maybe someone can test to confirm it is not the same as #20641

Tested with e9c083ad62b843e64d20ca04d05d0f3858016cfa

[New Game Projecte.zip](https://github.com/godotengine/godot/files/2270819/New.Game.Projecte.zip)

1. The sprite name is not the same
2. Run in Editor works fine
3. Run Exported Gamed(tested on x11, html5) shows only an error that test.png can't be found.

```
ERROR: load_image: Error opening file: res://sprites/test.png
   At: core/io/image_loader.cpp:55.
ERROR: texture_get_data: Condition ' !texture->active ' is true. returned: Ref<Image>()
   At: drivers/gles3/rasterizer_storage_gles3.cpp:1008.

```

Maybe it is related to https://github.com/godotengine/godot/issues/20641 but I don't see any name conflict.
I tried a few commits like ~1 or 2 months old, same issue. Feels like I am doing something wrong.
